### DISCUSSION TITLE
Support for Chinese spell checking

### DISCUSSION THREAD

**Discussion by peikai** [Zettlr/Zettlr#3253] [Ideas]
Hi,
Thanks for the contribution of this awesome software! I'm writing to express my expectation of support for Chinese spell checking. 
Is there any plan for it?

**Comment by nathanlesage** [comment]
Does the documentation of how to add new dictionaries to the spell checker not work for you?

**Reply by peikai** [reply]
I searched around, such as in [libreoffice dictionaries](https://cgit.freedesktop.org/libreoffice/dictionaries/tree/) and several github repos with most stars, but finally can not find the dict for Chinese Simplified (zh-CN).

**Reply by ycliu-pku** [reply]
I‘ve also run into this problem recently, so have you solved the problem?

**Reply by peikai** [reply]
I did not try afterwards. It still seems a missing function.

**Comment by exanrauzer** [comment]
In Beta Release you got LanguageTool integration.

**Reply by peikai** [reply]
Great news!
